src/tbb/governor.h

```diff
@@ -151,6 +158,68 @@ class governor {
         return false;
 #endif
     }
+
+#if __TBB_STATISTICS
+    enum class statistics : std::size_t {
+        exec_task = 0,
+        get_task,
+        task_res_not_null,
+        steal_task,
+        try_steal_task,
+        victim_pool_empty,
+        size,
+    };
+    using statistic_array = std::array<std::uint32_t, static_cast<std::size_t>(statistics::size)>;
+
+    class collect_statistics : no_copy {
+    public:
+        collect_statistics() = default;
+
+        ~collect_statistics() = default;
```

Comment:
Seems redundant

Reply:
We deleted copy/move constructors and copy/move assigned operators. Seems its ok